recipe/build.sh

```diff
@@ -33,9 +35,19 @@ cd ${SRC_DIR}
 #  rebuild static library using NAIF scripts
 export TKCOMPILER=${CC}
 cd ${SRC_DIR}/src/cspice
-${BUILD_PREFIX}/bin/tcsh ./mkprodct.csh
+if [ "$CI" == "drone" ];
+then
+    TKCOMPILEOPTIONS="-c -ansi -O2 -fPIC -DNON_UNIX_STDIO" TKLINKOPTIONS="-lm" ${BUILD_PREFIX}/bin/tcsh ./mkprodct.csh
```

Comment:
Why do you need this?

Reply:
The library provides a number of build scripts written in CSH that I didn't want to re-implement the logic for within the build.sh file. Specifically this line rebuilds the static library (ending in ".a") provided with the library zip archive on the CI worker to ensure they will work on the various target OSs.
Specifically, I have this call to remove the "-m64" parameter which is one of the default arguments CSPICE defines in the mkprodct.csh file which was causing build failures on the drone ARM builds (I was getting "unrecognized command line option" which can be seen in the logs https://cloud.drone.io/conda-forge/cspice-feedstock/10/1/2). I also need to do this below for the various executable programs CSPICE ships with to recompile them.

Reply:
I think patching cspice to remove `-m64` would make this cleaner.

Reply:
that is an option and it would certainly remove the if statements here and below... do you happen to know which selectors I would need to use to only apply the patch for drone ci builds? 

Another issue I see is that above the edit to use `${CFLAGS}` would  be good to pass in as well, maybe using the same variables I use here, just without the if statement.

Reply:
Yes, `-m64` should not be needed for any platform. So, you can apply the flags unconditionally.

Reply:
great! writing the patch now